### ISSUE TITLE
Nemesis issue

### ISSUE THREAD

**Issue by geophyzz** [Guitarninja2/thepath#13]
The battle scene between Calia in mist form with a bandit lord does not play in the Deus Ex Machina quest.  This seems to be a known issue with this animation and Nemesis: https://github.com/ShikyoKira/Project-New-Reign---Nemesis-Main/issues/436

**Comment by Guitarninja2** [comment]
Hmmmmm. That would be unfortunate if I need to remove nemesis.

**Comment by geophyzz** [comment]
If this was the only Enderal specific animation having an issue, it would seem like it might be better to stick with Nemesis and perhaps try to figure out the problem with that particular animation. But I think when I visited Jespar at the the watchtower he was supposed to be pipe smoking and that wasn't working either. So now I wonder if any Enderal specific animations are working.  I did some googling and came across this link:  https://www.reddit.com/r/enderal/comments/kvtkmg/enderal_how_to_setting_up_nemesis_behavior_engine/ .  Maybe using the FNIS Enderal patch mentioned in there will help?
